Frontend/ui-driver/src/Screens/EnterMobileNumberScreen/Controller.purs

```diff
@@ -77,7 +77,12 @@ eval (PrimaryEditTextAction (PrimaryEditText.TextChanged valId newVal)) state =
                                                     if newVal=="4000400040" || newVal=="3000300030" then true else false 
                                                         else true 
                                     Nothing -> true 
-  continue state { props = state.props { btnActive = if (length newVal == 10 && isValidMobileNumber) then true else false
+  let newState = state { props = state.props { btnActive = if (length newVal == 10 && isValidMobileNumber) then true else false
                                         , isValid = not isValidMobileNumber }
                                         , data = state.data { mobileNumber = if length newVal <= 10 then newVal else state.data.mobileNumber}}
+  continueWithCmd newState[
+    do
+        if(length newVal == 10 && isValidMobileNumber) then pure $ PrimaryButtonActionController (PrimaryButton.OnClick)
+          else pure $ AfterRender
+  ]
```

Comment:
no need to call `continueWithCmd`, you can directly call `exit (GoToNextScreen newState)` or `continue state` as per requirement

Reply:
same in other places also